src/internal/pachd/restore.go

```diff
@@ -26,9 +30,28 @@ func NewRestoreSnapshot(env RestoreSnapshotEnv, config pachconfig.PachdRestoreSn
 		printVersion(),
 		awaitDB(env.DB),
 		setupStep{
-			Name: "HelloWorld",
+			Name: "restore",
 			Fn: func(ctx context.Context) error {
-				log.Info(ctx, "Hello, you would have just restored a snapshot")
+				//tracker := track.NewPostgresTracker(env.DB)
```

Comment:
Looks like storage.New creates this for you, so yeah, I think it's safe to not include this.